.github/workflows/main.yml

```
@@ -0,0 +1,29 @@
+name: Build & Test
+
+on:
+  push:
+    branches: [trunk]
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [18.x, 20.x]
```

```suggestion
        node-version: [22.x, 23.x]
```

imo this should target LTS and latest, not deprecated versions like 18.x

---

we can't

the SQLite dependency we use fails to install under 22 and above

this is why I specifically left it at 18 and 20, until upstream sorts it out on their end

my mistake for not noting that down